src/send-query.js

```diff
@@ -0,0 +1,12 @@
+import ajax from './ajax';
+
+export default function sendQuery(query, config) {
+  const url = `https://${config.domain}/api/graph`;
+  const headers = {
+    authorization: `Basic ${btoa(config.apiKey)}`,
```

Comment:
Nope. The [isomorphic-btoa](https://github.com/Shopify/js-buy-sdk/blob/master/src/isomorphic-btoa.js) module adds it globally. An example of its usage as it works today can be seen [here](https://github.com/Shopify/js-buy-sdk/blob/master/src/adapters/listings-adapter.js#L13)